lambdas/create_student_user.py

```diff
@@ -11,53 +11,46 @@ def create_student_user(event, context):
 
     #Need to check if email is already in DB
-    check_email = client.execute_statement(
-        secretArn = "arn:aws:secretsmanager:us-east-2:500514381816:secret:rds-db-credentials/cluster-33FXTTBJUA6VTIJBXQWHEGXQRE/postgres-3QyWu7",
-        database = "postgres",
-        resourceArn = "arn:aws:rds:us-east-2:500514381816:cluster:postgres",
-        sql = "SELECT email FROM users WHERE email = '%s';" % (email)
-    )
+    sql = "SELECT email FROM users WHERE email = :email"
+    sql_parameters = [{'name' : 'email', 'value': {'stringValue' : email}}]
+    check_email = query(sql, sql_parameters)
 
     if(check_email['records'] != []):
         return{
             'body': json.dumps("email is already being used"),
             'statusCode': 409
         }
     # generates a new id 
-    new_id_query = client.execute_statement(
-        secretArn = "arn:aws:secretsmanager:us-east-2:500514381816:secret:rds-db-credentials/cluster-33FXTTBJUA6VTIJBXQWHEGXQRE/postgres-3QyWu7",
-        database = "postgres",
-        resourceArn = "arn:aws:rds:us-east-2:500514381816:cluster:postgres",
-        sql = "SELECT id FROM users ORDER BY id DESC LIMIT 1;"
-    )
+    sql = "SELECT id FROM users ORDER BY id DESC LIMIT 1"
+    sql_parameters = []
+    new_id_query = query(sql,sql_parameters)
     new_id = new_id_query['records'][0][0]['longValue'] + 1
 
     # adds a user to the "users" table with the new generated id
-    create_users_instance = client.execute_statement(
-        secretArn = "arn:aws:secretsmanager:us-east-2:500514381816:secret:rds-db-credentials/cluster-33FXTTBJUA6VTIJBXQWHEGXQRE/postgres-3QyWu7",
-        database = "postgres",
-        resourceArn = "arn:aws:rds:us-east-2:500514381816:cluster:postgres",
-        sql = "INSERT INTO users(id,first_name,last_name, email, preferred_name, picture, bio,pronouns,gender,phone,is_blocked,GCal_permission,hashed_password,salt_key,user_type) \
-        VALUES ('%s','%s','%s','%s','pn','pic','bio','pro','gen','pho',false,true,'%s','salt','student')" % (new_id,first_name,last_name,email,password)
-    )
+    sql = """INSERT INTO users(id,first_name,last_name, email, preferred_name, picture, bio,pronouns,gender,phone,is_blocked,GCal_permission,hashed_password,salt_key,user_type) \
+        VALUES (:new_id,:first_name,:last_name,:email,'pn','pic','bio','pro','gen','pho',false,true,:password,'salt','student')"""
+    sql_parameters = [{'name' : 'new_id', 'value': {'longValue' : new_id}},
+    {'name' : 'first_name', 'value': {'stringValue' : first_name}},
+    {'name' : 'last_name', 'value': {'stringValue' : last_name}},
+    {'name' : 'email', 'value': {'stringValue' : email}},
+    {'name' : 'password', 'value': {'stringValue' : password}}]
+
+    create_users_instance = query(sql,sql_parameters)
 
     if(create_users_instance['numberOfRecordsUpdated'] == 0): 
         return {
             'body': json.dumps("Student user not created"),
-            'statusCode': 5
+            'statusCode': 500
```

Comment:
You should add the error message returned from query_db to your ouput, so it will be logged. This will help with debugging